ue5osc/osc_dispatcher.py

```diff
@@ -0,0 +1,104 @@
+from pythonosc.dispatcher import Dispatcher
+from typing import List, Any
+from pythonosc.osc_server import BlockingOSCUDPServer
+from pythonosc.udp_client import SimpleUDPClient
+from time import sleep
+
+# import threading
+
+# functions inside dispatcher to clean up the code
+# request/save image
+
+
+class DispatchHandler:
+    def __init__(self):
+        # Initialize attributes
+        self.location_values = None
+        self.rotation_values = None
+        self.project_name = None
```

Comment:
We might be able to use a single value variable, which could make the `wait`/`delay` function simpler.